recipes/newrelic/apm/php/php-install.yml

```diff
@@ -0,0 +1,27 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: php-agent-installer
+displayName: PHP Agent Installer
+description: New Relic install recipe for instrumenting PHP applications
+repository: https://github.com/newrelic/newrelic-php-agent
+
+installTargets:
+  - type: application
+    os: linux
+  - type: application
```

Comment:
I dont think there is a PHP on windows? I know we dont have an agent for it

Reply:
You can certainly run PHP on Windows, however, I did look it up, and you're correct, we don't have a PHP Agent on Windows!

Removing recipe for this.